spec/forms/steps/client/has_nino_form_spec.rb

```diff
@@ -55,20 +52,6 @@
       end
     end
 
-    context 'when `has_nino` is valid' do
```

Comment:
@willmcb I removed this, as in reality is the same test the shared examples perform below in the 
```it_behaves_like 'a has-one-association form'``` as it is the happy path.